frontend/ui/quickstart.lua

```diff
@@ -46,7 +47,27 @@ div.table > div > *:nth-child(2) { text-align: left; hyphens: none; background-c
 div.table > div > *:nth-child(3) { white-space: nowrap; }
 ]]
 
-local quickstart_guide = T(_([[
+local quickstart_guide
+if Device:hasScreenKB() or Device:hasSymKey() then
+    quickstart_guide = T(_([[
```

Comment:
Should be closed by  ]] and ) and ).
On line 69, I see:
```lua
]])
else
```
The T is supposed to take arguments, to replace %1, %2... with them.
I see it was at the end:
```lua
<div class="generated">Generated by KOReader %1.</div>
]]),
    rev)
```
So, either you don't need T, or you forgot that `, rev)` in some of them.